virt/virt-4-10-release-notes.adoc

```diff
@@ -0,0 +1,228 @@
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+
+[id="virt-4-10-new"]
+== New and changed features
+
+* OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS__.
+** Intel and AMD CPUs.
+
+//CNV-13851 SVVP for 4.10: Ensure platform passes Windows Server Virtualization Validation Program - with RHCOS workers
+
+//CNV-13807 Release readiness and closure
+
+[id="virt-4-10-quick-starts"]
+=== Quick starts
+
+* Quick start tours are available for several {VirtProductName} features. To view the tours, click the *Help* icon *?* in the menu bar on the header of the {VirtProductName} console and then select *Quick Starts*. You can filter the available tours by entering the `virtualization` keyword in the *Filter* field.
+
+[id="virt-4-10-installation-new"]
+=== Installation
+
+//CNV-14192 Update launchers using live-migration - default/opt out
+
+//CNV-14910 HyperConverged Operator technical debt
+
+[id="virt-4-10-networking-new"]
+=== Networking
+
+//CNV-11201 Rollout of NodeNetworkConfigurationPolicy
+
+//CNV-13660 Inherit static IP from a NIC attached to the bridge
+
+//CNV-13679 SR-IOV live-migration without privilege escalation
+
+//CNV-13680 Primary and secondary networks over a shared NIC on OVN clusters
+
+//CNV-8577 CNV-14481 Service Mesh for VMs on the primary network with IPv4 (Done in CNV-8577 for 4.9 then reverted in CNV-14481)
+
+[id="virt-4-10-storage-new"]
+=== Storage
+
+[id="virt-4-10-web-new"]
+=== Web console
+
+// NOTE: no deprecated features in 4.10; commenting out these sections to retain for future use
+//[id="virt-4-10-deprecated-removed"]
+//== Deprecated and removed features
+
+//[id="virt-4-10-deprecated"]
+//=== Deprecated features
+
+//Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
+
+// NOTE: when uncommenting deprecated features list, change the header level below to ===
+
+//[id="virt-4-10-removed"]
+//=== Removed features
+
+//Removed features are not supported in the current release.
+
+// NOTE: no notable technical changes in 4.9, commenting out to retain
+//[id="virt-4-10-changes"]
+//== Notable technical changes
+
+[id="virt-4-10-technology-preview"]
+== Technology Preview features
+
```

Comment:
We'll need to check to be sure that TP still applies to these features. For example, `//CNV-14192 virt-launcher now automatically updates - now TP` should be removed for 4.10. I'll be adding a RN to the installation section to replace this.

Reply:
Removed the CNV-14192 bullet, sent an email to check on the rest of these features. Thanks Pan!

Reply:
According to @aglitke Hot-plug is planned for GA in 4.10, so I've removed this TP note as well. I will work with @ctomasko to create release note Jiras for these TP features for insertion in the New and Changed sections.